src/Sylius/Component/Order/Model/Order.php

```diff
@@ -435,7 +435,9 @@ public function getAdjustmentsTotal($type = null)
 
         $total = 0;
         foreach ($this->getAdjustments($type) as $adjustment) {
-            $total += $adjustment->getAmount();
+            if (!$adjustment->isNeutral()) {
+                $total += $adjustment->getAmount();
+            }
```

Comment:
The only problem with simple `$this->getAdjustmentsTotal()` is when you add your custom adjustment e.g. `COMMISION_FEE` and apply it on `Order` then it will be included in shipping total, but with:
```
$shippingTotal = $this->getAdjustmentsTotal(AdjustmentInterface::SHIPPING_ADJUSTMENT);
$shippingTotal += $this->getAdjustmentsTotal(AdjustmentInterface::ORDER_PROMOTION_ADJUSTMENT);
$shippingTotal += $this->getAdjustmentsTotal(AdjustmentInterface::TAX_ADJUSTMENT);
```
It will not be included. There is already big assumption that `TAX` and `ORDER_PROMOTION` adjustments belong to shipping, so we should at least not limit the above use case.

However we can write a short note in the docs that if you want to add your own adjustment on `Order` change implementation of `getShippingTotal` logic to "this". Worse customization DX / little implementation overhead, we have to choose 😄  @pjedrzejewski any thoughts on this?

I like the idea to return `adjustmentsTotal` as shipping total since it even does not require to fetch adjustments collection or calculate anything :)